cluster-autoscaler/core/filter_out_schedulable.go

```diff
@@ -67,20 +65,11 @@ func (filterOutSchedulablePodListProcessor) Process(
 	filterOutSchedulableStart := time.Now()
 	var unschedulablePodsToHelp []*apiv1.Pod
 
-	if context.EstimatorName == estimator.BinpackingEstimatorName {
-		unschedulablePodsToHelp = filterOutSchedulableByPacking(unschedulablePods, upcomingNodes, allScheduledPods,
-			context.PredicateChecker, context.ExpendablePodsPriorityCutoff, false)
-	} else {
-		unschedulablePodsToHelp = unschedulablePods
-	}
+	unschedulablePodsToHelp = filterOutSchedulableByPacking(unschedulablePods, upcomingNodes, allScheduledPods,
```

Comment:
Add some comment here. It looks really strange that we're calling this twice like that and if I didn't already knew how it works, I imagine I'd spent a while figuring it out.

Reply:
I will. It gets refactored with ClusterSnapshot anyway :)